### DISCUSSION TITLE
Time to learn how to make a LORA

### DISCUSSION THREAD

**Discussion by JoelHJ** [lllyasviel/Fooocus#1721] [Q&A]
Hi
So have been using Foocus now for a while and really want to try to make my own LORA. Have been looking at youtube and Google and there seems to be a million and one way. Why not ask in this community what you think is the best/easiest way to start. Happy to spend some funds if need be. I have a RTX3070 Ti not sure if that is enough to try to do it locally or if i rent something like Runpod with Stable Diffusion Kohya_ss ComfyUI Ultimate setup. Any ideas as to how to start would be great.

Thank you
J

**Comment by mashb1t** [comment]
Kohya_ss and Runpod will do best, 8GB VRAM aren't enough for (fast) training SDXL LoRAs (i had 42h for a SDXL LoRA with 3800 steps in total when using a 3080 10GB VRAM as it started outsourcing to RAM). EDIT: this is why i stopped local training and moved to a friend of mine with a 4090, which only took 1h 20min afaik to generate it with my config.
There are pretty good tutorials available in https://github.com/bmaltais/kohya_ss?tab=readme-ov-file#tutorials, also a [template for Runpod](https://github.com/bmaltais/kohya_ss?tab=readme-ov-file#pre-built-runpod-template).
Some insights: the LoRA used for https://gallery.mashb1t.de/gallery/w895oePvAULKFAg_J8tKRLFP is based on 19 images with each 20 steps and 10 epochs using adafactory (as described in the tutorial).
All the best of luck for you and if you want to you can shoot some results here!

**Reply by JoelHJ** [reply]
Thank you so much for your reply. I will hop over and check it all out now. Thank you again..
